src/profile/v100/profile.rs

```diff
@@ -525,6 +535,77 @@ mod tests {
         );
     }
 
+    #[test]
+    fn check_if_profile_json_contains_legacy_p2p_links_when_p2p_links_are_present(
+    ) {
+        let value = r#"
+        {
+            "appPreferences": {
+              "p2pLinks": [
+                {
+                  "connectionPassword": "babebabebabebabebabebabebabebabebabebabebabebabebabebabebabebabe",
+                  "displayName": "Brave on PC"
+                }
+              ]
+            }
+          }
+        "#;
+        assert_eq!(
+            true,
```

Comment:
please flip the args, I have been using RHS as "expected value" throughout the whole of Rust Sargon.

Reply:
for this and all your other `assert` ... :)